com.jeffser.Alpaca.AMD.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "com.jeffser.Alpaca.AMD",
+    "branch": "1.0",
+    "runtime": "com.jeffser.Alpaca",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk//46",
+    "build-extension": true,
+    "build-options": {
+        "prefix": "/app"
+    },
+    "cleanup": [
+        "*.a",
+        "*.la"
+    ],
+    "modules": [
+        {
+            "name": "rocm",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -r --remove-destination lib/ollama/* ${FLATPAK_DEST}/"
+            ],
+            "only-arches": [
+                "x86_64"
+            ],
```

Comment:
needs flathub.json too https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson